python/erdos/message.py

```diff
@@ -55,8 +55,16 @@ def _to_py_message(self) -> PyMessage:
             self._serialize_data()
         return PyMessage(self.timestamp._to_py_timestamp(), self._serialized_data)
 
+    def __repr__(self) -> str:
+        return f"Message({self.timestamp}, {self.data})"
+
     def __str__(self) -> str:
-        return "{{timestamp: {}, data: {}}}".format(self.timestamp, self.data)
+        return f"{{timestamp: {self.timestamp}, data: {self.data}}}"
+
+    def __eq__(self, msg: object) -> bool:
+        if not isinstance(msg, Message):
+            raise ValueError(f"Equality with '{type(msg)}' is not implemented.")
+        return self.timestamp == msg.timestamp and self.data == msg.data
```

Comment:
Can you provide some potential use-cases for checking equality of messages?

Reply:
Sure. In Pylot, I'm refactoring an operator and need to check the messages from 2 streams are identical.